examples/test/laser-controls/alignment.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Laser Controls - Alignment</title>
+    <meta name="description" content="Raycaster - A-Frame">
+    <script src="../../../dist/aframe-master.js"></script>
+  </head>
+  <body>
+    <a-scene>
+      <a-entity position="-2 1.5 -2">
+        <a-entity vive-controls cursor="laserEnabled: true" raycaster="objects: null"></a-entity>
+      </a-entity>
+
+      <a-entity position="-1 1.5 -2">
+        <a-entity oculus-touch-controls cursor="laserEnabled: true" raycaster="far: 2; objects: null; origin: 0 0 0.1; direction: 0 -0.4 -1"></a-entity>
+      </a-entity>
+
+      <a-entity position="0 1.5 -2">
+        <a-entity daydream-controls cursor="laserEnabled: true" raycaster="objects: null"></a-entity>
+      </a-entity>
+
+      <a-entity position="1 1.5 -2">
+        <a-entity gearvr-controls cursor="laserEnabled: true" raycaster="objects: null; origin: 0 0.005 0"></a-entity>
```

Comment:
I notice there's `objects: null` quite a bit; perhaps the [default](https://github.com/aframevr/aframe/pull/2678/files#diff-d0074fac66c607708186057c046f5dbeR25) should change based on `cursor="laserEnabled: true"`?